### PR TITLE
Attendees endpoint init

### PR DESCRIPTION
**Pull request by LayanJay**
## Purpose

The purpose of this PR is to fix #<issue-number>

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have performed a self-review of my own code
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

**Comment by Shehanka** [comment]
@LayanJay can u resolve the conflict

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+15 info attempt registry request try #1 at 10:00:01 PM
+16 verbose request id 8058916795d20d8c
+17 http request GET https://registry.npmjs.org/yarn
+18 http 200 https://registry.npmjs.org/yarn
+19 verbose headers { date: 'Sun, 28 Feb 2021 16:30:01 GMT',
+19 verbose headers   'content-type': 'application/json',
+19 verbose headers   'transfer-encoding': 'chunked',
+19 verbose headers   connection: 'keep-alive',
+19 verbose headers   'set-cookie':
+19 verbose headers    [ '__cfduid=dc3d9cdc625570a40445439c01e04364d1614529800; expires=Tue, 30-Mar-21 16:30:00 GMT; path=/; domain=.npmjs.org; HttpOnly; SameSite=Lax' ],
```
Comment (Shehanka) [inline npm-debug.log:25]: gitignore log file

**Review by Shehanka — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,7 @@ node_modules
 build
 logs
 .idea/
+
+.env*
+
```
Comment (ebonynon) [inline .gitignore:7]: Add yarn.lock file

File: src/api/attendees/attendee.controller.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ *
+ * @param req
+ * @param res
+ * @returns {Promise<*>}
+ */
+
+const createAttendee = async (req, res) => {
+  try {
+    const attendee = await AttendeeService.createAttendee(req.body);
```
Comment (ebonynon) [inline src/api/attendees/attendee.controller.js:13]: Set logs `logger.info(....)`

File: src/api/attendees/attendee.controller.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    return res.status(HTTP_STATUS.CREATED).json(attendee);
+  } catch (err) {
+    return res
+      .status(HTTP_STATUS.INTERNAL_SERVER_ERROR)
```
Comment (ebonynon) [inline src/api/attendees/attendee.controller.js:17]: Set error logs `logger.error(....)`

File: src/api/attendees/attendee.modal.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    email: { type: String, required: true },
+    contactNo: { type: String, required: true },
+    previlege: { type: String, required: true },
+  },
+  { timestamps: true }
+);
+
+export default model("Attendees", attendeeSchema);
```
Comment (ebonynon) [inline src/api/attendees/attendee.modal.js:18]: Change `"Attendees"` as `"attendees"`

File: src/api/attendees/attendee.modal.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import { Schema, model } from "mongoose";
+
+/**
+ *
+ * @type {module:mongoose.Schema<any>}
+ */
+const attendeeSchema = new Schema(
+  {
+    firstName: { type: String, required: true },
```
Comment (ebonynon) [inline src/api/attendees/attendee.modal.js:9]: Make as unique and lowercase  `unique: true, lowercase: true`

**Review by ebonynon — COMMENTED** [review]
Set info and error logs. Please remove yarn.lock

**Review by LayanJay — COMMENTED** [review]
Resolved

**Pull request merged**